plans/specs/PRODUCTION_MVP_SPEC.md

```
@@ -0,0 +1,1067 @@
+name = "celery-worker"
+startCommand = "celery -A src.celery_app worker --loglevel=info --concurrency=2"
+
+[[services]]
+name = "telegram-bot"
+startCommand = "python src/telegram_bot/bot.py"
+```
+
+### 10.2 Environment Variables
+
+**Backend:**
+```bash
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)